### PR TITLE
Add caption kirknwhite

### PR DESCRIPTION
**Pull request by kirknwhite**
## Pull Request
This PR Updates the caption because it was out of date
This PR closed / resolves / fixes #2
 
## Done
- Created a branch
- Made a commit
- 
## To do
- [ ]  merge the branch
- [ ] delete the ## My Pull Request
 
@omoferacho

### CODE REVIEW

File: _slides/12-kirknwhite.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+"This is my first day of GITHUB training second time"
```
Comment (RileyBurke) [inline _slides/12-kirknwhite.md:6]: Review comment

**Review by RileyBurke — APPROVED** [review]
Approved.

File: _slides/12-kirknwhite.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (clmackey) [inline _slides/12-kirknwhite.md:3]: Nice work.  You da man!

**Review by clmackey — APPROVED** [review]
Good to go!

**Pull request merged**